pwiz_tools/Shared/ProteowizardWrapper/MsDataFileImpl.cs

```diff
@@ -612,19 +612,59 @@ public double[] GetTotalIonCurrent()
             }
             using (var chromatogram = ChromatogramList.chromatogram(0, true))
             {
-                if (chromatogram == null)
+                return chromatogram?.getIntensityArray()?.data.Storage();
+            }
+        }
+
+        public class QcTrace
+        {
+            public QcTrace(Chromatogram c)
+            {
+                Name = c.id;
+                if (Name.ToLowerInvariant().Contains(@"pressure"))

```

Comment:
I guess okay, but any more of these and we'll want a dictionary at the top and look-up instead of adding more ifs.

Reply:
When the MS CV is updated to add chromatogram types for pressure and flow rate, I will use that term as the if condition rather than the string matching. But I'd still plan to use if-blocks because I don't plan to re-use this logic anywhere else (indeed, the point of adding these properties was to avoid the need for any downstream code to do its own lookup other than translation). Is there another reason to use a dictionary/mapping in this case other than code-reuse? I assume you mean something like:
```
Dictionary<string, (string Quantity, string Units)> lookup = new Dictionary<string, (string, string)>()
{
   {"pressure", ("pressure", "psi") },
   {"flow", ("volumetric flow rate", "uL/min") }
}
```

As described above I would really imagining this to end up as:
```
Dictionary<CVID, (QcQuantity Quantity, QcUnits Units)> lookup = new Dictionary<CVID, (QcQuantity, QcUnits)>()
{
   {CVID.MS_pressure_trace, (QcQuantity.Pressure, QcUnits.PoundsPerSquareInch) },
   {CVID.MS_flow_rate_trace, (QcQuantity.VolumetricFlowRate, QcUnits.MicrolitersPerMinute") }
}
```
(QC quantity and units would also be in the CVID enum, but I don't want to force Skyline to import the CV since it doesn't currently do so)